adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Telemetry.cs

```diff
@@ -68,5 +68,14 @@ public void StopEvent(string requestId, EventBase eventToStop)
         {
             
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="requestId"></param>
```

Comment:
We tend to remove empty comments and / or empty comment sections.

Reply:
Added a comment here like in the others.